scripts/cat/enums/status.py

```diff
@@ -0,0 +1,157 @@
+from scripts.game_structure.extended_strenum import ExtendedStrEnum
+
+
+class StatusEnum(ExtendedStrEnum):
+    """
+    Roles that a cat may hold. Check ExtendedStrEnum for more things!
```

Comment:
Extremely nitpicky here, but is the "Check […]" part necessary?

Reply:
Honestly no, but it's there as a prompt to newer developers that there are other things you can do with this class that aren't right there. I don't know if that's actually necessary or not in honesty haha, I just like how friendly the codebase is for beginners. I don't want my refactors to inhibit that nature, even if they're inherently making things more complicated.

Reply:
Good enough for me. Keep it in then!